packages/snack-sdk/src/transports/TransportImplWebPlayer.ts

```diff
@@ -0,0 +1,79 @@
+import assert from 'assert';
+
+import type { SnackWindowRef } from '../types';
+import type { ProtocolOutgoingMessage, ProtocolCodeMessage } from './Protocol';
+import TransportImplBase from './TransportImplBase';
+
+export interface TransportWebPlayerConfig {
+  name?: string;
+  verbose?: boolean;
+  webPlayerURL: string;
+  window: Window;
+  ref: SnackWindowRef;
+}
+
+export default class TransportImplWebPlayer extends TransportImplBase {
+  private _currentWindow: Window | undefined;
+  private _targetWindowRef: SnackWindowRef | undefined;
+  private _isStarted: boolean;
+  private readonly _origin: string;
```

Comment:
Same here :)

> Another personal preference, feel free to ignore. But with TypeScript, we can already define private/public/protected, we don't need to do that in the name anymore :)

```suggestion
  private currentWindow: Window | undefined;
  private targetWindowRef: SnackWindowRef | undefined;
  private isStarted: boolean;
  private readonly origin: string;
```

Reply:
Also, random sanity check, is `started` the only status this transport can have? If so, ignore this suggestion, if there are more, maybe this would be more appropriate.

```suggestion
  private _currentWindow: Window | undefined;
  private _targetWindowRef: SnackWindowRef | undefined;
  private status: 'stopped' | 'started' = 'stopped';
  private readonly _origin: string;
```